### PR TITLE
Update UI with new changes in content APIs

### PR DESCRIPTION
**Pull request by shubham251972**
**Notes for Reviewers**

This PR fixes #

This PR updates UI according to new changes introduced in content APIs

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by shubham251972** [comment]
This PR depends on cloud release, please do not merge

File: ui/components/configuratorComponents/MeshModel/hooks/useDesignLifecycle.js [hunk 1]
```diff
@@ -73,7 +73,8 @@ export default function useDesignLifecycle() {
       method: 'POST',
     })
       .then((data) => {
-        setDesignId(data[0].id);
+        // Index zero here will be valid as response will always contain single design
```
Comment (leecalcote) [inline ui/components/configuratorComponents/MeshModel/hooks/useDesignLifecycle.js:76]: Thank you for including this explanation, @shubham251972.

**Review by leecalcote — APPROVED** [review]

**Comment by jameshorton2337** [comment]
> This PR depends on cloud release, please do not merge

Has that Cloud release been made?

**Comment by sudhanshutech** [comment]
merge conflicts @shubham251972

**Comment by leecalcote** [comment]
Merge conflicts. @shubham251972 

@Yashsharma1911

**Comment by Yashsharma1911** [comment]
> Merge conflicts. @shubham251972
> 
> @Yashsharma1911

This PR should only merge if we have merged my content PR in cloud otherwise it will break changes, if not let's close this PR

**Comment by leecalcote** [comment]
Screenshot 2024-09-12 at 4 34 13 PM
This is the PR, I think....

**Pull request closed without merging**